### PR TITLE
change from housepower to clickhouse

### PR DESCRIPTION
**Pull request by mzitnik**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -62,3 +62,5 @@ kyuubi_version=1.9.0
 testcontainers_scala_version=0.41.2
 scalatest_version=3.2.16
 flexmark_version=0.62.2
+
```
Comment (pan3793) [inline gradle.properties:65]: nit: unnecessary changes.

File: gradle.properties [hunk 2]
```diff
@@ -13,15 +13,15 @@
 #
 
 mavenCentralMirror=https://mirrors.cloud.tencent.com/nexus/repository/maven-public/
```
Comment (pan3793) [inline gradle.properties:15]: Gradle does not have a built-in Repo Mirror setting like Maven does, we have to explicitly expose a variable. We can keep it with the default value `https://repo1.maven.org/maven2/`. Users can override this variable in `$HOME/.gradle/gradle.properties`

File: build.gradle [hunk 3]
```diff
@@ -68,7 +68,7 @@ allprojects {
     version = getProjectVersion()
 
     repositories {
-        maven { url = "$mavenCentralMirror" }
+        mavenCentral()
```
Comment (pan3793) [inline build.gradle:71]: ditto, would like to keep it

File: .github/workflows/publish-snapshot.yml [hunk 4]
```diff
@@ -45,4 +45,3 @@ jobs:
           -Dspark_binary_version=${{ matrix.spark }}
           -PmavenUser='${{ secrets.NEXUS_USER }}'
           -PmavenPassword='${{ secrets.NEXUS_PW }}'
```
Comment (pan3793) [inline .github/workflows/publish-snapshot.yml:47]: I suppose you have set these secrets properly, right?

File: .github/workflows/publish-release.yml [hunk 5]
```diff
@@ -51,10 +51,9 @@ jobs:
           -Prelease
           -Psigning.gnupg.keyName='Cheng Pan'
```
Comment (pan3793) [inline .github/workflows/publish-release.yml:52]: a new PGP key should be generated and set properly

**Review by Old177 — CHANGES_REQUESTED** [review]
Leav

**Review by Old177 — CHANGES_REQUESTED** [review]
The

**Comment by pan3793** [comment]
@Old177 do you have any questions? please stop creating spam comments/issues

**Review by Old177 — CHANGES_REQUESTED** [review]
U

File: gradle.properties [hunk 6]
```diff
@@ -62,3 +62,5 @@ kyuubi_version=1.9.0
 testcontainers_scala_version=0.41.2
 scalatest_version=3.2.16
 flexmark_version=0.62.2
+
+
```
Comment (pan3793) [inline gradle.properties:66]:
```suggestion
```

File: build.gradle (hunk 3 above)
Comment (pan3793) [inline build.gradle:71]:
```suggestion
        maven { url = "$mavenCentralMirror" }
```

**Review by pan3793 — APPROVED** [review]
I'm fine with the existing changes, but I suppose additional changes are required to make the release happy.

**Review by mshustov — APPROVED** [review]

**Pull request merged**